cookbook/images/network-image.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: "Display images from the internet"
+permalink: /cookbook/images/network-image/
+---
+
+Displaying images is fundamental for most mobile apps. Flutter provides the 
+[`Image`](https://docs.flutter.io/flutter/widgets/Image-class.html) Widget to 
+display different types of images.
+
+In order to work with images from the web, use the [`Image.network`](https://docs.flutter.io/flutter/widgets/Image/Image.network.html) 
+constructor.
+
+```dart
+new Image.network(
+  'https://github.com/flutter/website/blob/master/_includes/code/layout/lakes/images/lake.jpg?raw=true',
+)
+```
+
+## Bonus: Animated Gifs
```

Comment:
would this be a separate recipe?

Reply:
I could easily make it a separate recipe, but since it uses the exact same Widget / constructor, I figured it was more of a "Bonus point" than separate recipe.

Reply:
+1 for leaving that here for the moment; we can revisit areas that we feel deserve their own recipe later on